components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2649,6 +2649,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         const context = (await this.contextParser.handle(ctx, user, contextURL)) as CommitContext;
 
+        // HACK: treat manual triggered prebuild as a reset for the inactivity state
```

Comment:
This doesn't seem like a hack to me 👍 (especially if we rename `lastWorkspaceStart` to something more accurate like `lastUserActivity`)

```suggestion
        // Treat manual triggered prebuild as a reset for the inactivity state
```